apprise/plugins/NotifyEmail.py

```diff
@@ -788,15 +797,21 @@ def send(self, body, title='', notify_type=NotifyType.INFO, attach=None,
                     (from_name if from_name else False, self.from_addr))
                 base['To'] = formataddr((to_name, to_addr))
 
-            base['Message-ID'] = make_msgid(domain=self.smtp_host)
+            try:
+                base['Message-ID'] = make_msgid(domain=self.smtp_host)
+
+            except TypeError:
+                # Python v2.x Support (no domain keyword)
+                base['Message-ID'] = make_msgid()
+
             base['Date'] = \
                 datetime.utcnow().strftime("%a, %d %b %Y %H:%M:%S +0000")
             base['X-Application'] = self.app_id
 
             if cc:
                 base['Cc'] = ','.join(cc)
-            if reply_to:
-                base['Reply-To'] = ','.join(reply_to)
+
+            base['Reply-To'] = ','.join(reply_to)
```

Comment:
Why removing the condition? The Reply-To header will be sent even empty...
I also add the cc condition for the same reason.

Reply:
I love it that the tables completely reversed and i attempted to make the code better and failed.   :laughing: .  No worries here, we can revert.  I was just going for consistency.  By having the `set()` always populated, a lof of the if code drops and it's easy to handle it in the `def url()` function.

But if it actually makes no sense at all to have it can be undone for sure.

Reply:
I don't well understand what do you try to do, if you prefer another implementation, no problem at all :) The only thing is that the Reply-To header must be added to the email only if a value is set.